### DISCUSSION TITLE
Tools not opening (command not found)

### DISCUSSION THREAD

**Discussion by Airlegendz** [BlackArch/blackarch#4177] [Q&A] [answered]
I downloaded black arch over arch but when I try to use my installed tools (example: sudo skydive) it is not opening it and it is returning sudo: (app name): command not found even though i installed them

I installed black arch with the following commands

$ curl -O https://blackarch.org/strap.sh
$ echo 26849980b35a42e6e192c6d9ed8c46f0d6d06047 strap.sh | sha1sum -c
$ chmod +x strap.sh
$ sudo ./strap.sh
$ sudo pacman -Syu
$ sudo pacman -S blackarch

Update: I got it fixed, it was a problem with multilib

**Comment by anunna** [comment, marked as answer]
If you did the one thing that is not recommended and warned about on the BlackArch website; I can expect this. 

If you run: `pacman -S blackarch`, you can expect many problems. Remember you are adding a repository no different from official `core`, `extra`, `multilib`, etc. Installing the entire repository is not installing BlackArch the same as installing the entire Arch Linux `core`, `extra`, `multilib` repositories is not installing Arch Linux. 

If you run for example 
```
which skydive
```
```
> which skydive
/usr/sbin/skydive
```
or
```
pacman -Qe | grep skydive
```
```
> pacman -Qe | grep skydive
skydive 0.28.0-1
skydive-debug 0.28.0-1
```
Do you see it installed? I tested installing skydive to test any package issue and I had no installation or launch issues. 

```
> skydive
Skydive

Usage:
  skydive [command]

Available Commands:
  agent           Skydive agent
  allinone        Skydive All-In-One
  analyzer        Skydive analyzer
  bash-completion Generate bash completion helper
  client          Skydive client
  help            Help about any command
  inject-packet   inject packet
  seed            Skydive seed
  version         Print the version number of Skydive

Flags:
  -c, --conf stringArray        location of Skydive configuration files, default try loading /etc/skydive/skydive.yml if exist
  -b, --config-backend string   configuration backend (defaults to file) (default "file")
  -h, --help                    help for skydive
      --verbose                 verbose mode

Use "skydive [command] --help" for more information about a command.
```
